keyboards/kbdfans/kbdpad_mk2/keymaps/iracing_replay/keymap.c

```diff
@@ -0,0 +1,177 @@
+#include QMK_KEYBOARD_H
+
+void tap_key(uint16_t keycode) {
+  register_code  (keycode);
+  unregister_code(keycode);
+}
+
+void shift_key(uint16_t keycode) {
+  register_code  (KC_LSFT);
+  tap_key        (keycode);
+  unregister_code(KC_LSFT);
+}
```

Comment:
These functions already exist as `tap_code` and `tap_code16`.

Though, for `tap_code16`, you'd want `tap_code16(LSFT(kc))`

```suggestion
```

Reply:
Glad to find those.  Fixed.